August 2026 newsletter (#1067)
RSE-Sheffield/RSE-Sheffield.github.io: _newsletters/2026-08-03-newsletter.md

---
layout: post
title: August 2026 Newsletter
editor: Michael Foster
slug: 2026-08-03-newsletter
date: 2026-08-03T12:00:00Z+0100
tags:
category:
link:
description:
type: text
---

## The University of Sheffield Research Software Engineering Community Newsletter August 2026

Welcome to the August 2026 newsletter for the research software community at The University of Sheffield, featuring
news, opportunities, events and training for you.

<!-- --- -->

### News

> - [RSEcon](https://rsecon26.society-rse.org/) 2026 will be in Sheffield **2026-09-09 - 2026-09-11**. Join us in The Wave to celebrate a decade of Research Software Engineering! The draft programme is available [here](https://virtual.oxfordabstracts.com/e/rsecon26/program).

Reviewer: [nit]
```suggestion
- [RSECon26](https://rsecon26.society-rse.org/) will be in Sheffield **2026-09-09 - 2026-09-11**. Join us in The Wave to celebrate a decade of Research Software Engineering! The draft programme is available [here](https://virtual.oxfordabstracts.com/e/rsecon26/program).
```

  - [Registration](https://rsecon26.society-rse.org/registration/) deadline: 2026-08-07 (in person), 2026-08-31 (online)

---

### Events

<!-- Nothing scheduled but left as a place-holder should this be used as a template for next newsletter -->

<!-- ### RSE lunchtime seminars -->

<!-- Members of Sheffield RSE will be hosting a new small seminar series in the new year, with **catered lunches**! -->

<!-- You can sign up on [MyDevelopment]() (requires sign in to MUSE). -->

#### Upcoming Events in Sheffield

- [2026 Geoscience RSE SIG Annual Meeting](https://socrse.github.io/geoscience-sig/)
  - Challenges and Opportunities of RSE Careers in Geosciences. We are calling for lightning talks (5 mins) from the community to highlight recent work, advertise a talk, event or tool. If you're interested, please [sign up](https://forms.gle/ABCeAWwUv9aEQLQ86) by 16th August.

>   - Location: Co-located with [RSEcon](https://rsecon26.society-rse.org/)

Reviewer: [nit]
```suggestion
  - Location: Co-located with [RSECon26](https://rsecon26.society-rse.org/)
```

- [International Research Software Conference](https://www.researchsoft.org/irsc/program/)
  -  IRSC brings together leaders and decision-makers to shape the future of research software in the age of generative AI.
  - Date: **2026-09-07 - 2026-09-08**

>   - Location: Co-located with [RSEcon](https://rsecon26.society-rse.org/)

Reviewer: ```suggestion
  - Location: Co-located with [RSECon26](https://rsecon26.society-rse.org/)
```

- [Digital Humanities Congress 2026](https://www.dhi.ac.uk/congress)

>   - Date: **2026-09-17**

Reviewer 2: ```suggestion
  - Date: **2026-09-02 - 2026-09-03**
```

This sub-section will need moving above the previous item, so things remain in chronological order.

  - Registration deadline: **2026-08-17**

>   - The University of Sheffield's Digital Humanities Institute is delighted to announce that its two-day conference will be held in Sheffield on Wednesday 2nd and Thursday 3rd September 2026.

Reviewer: Date in line 50 text is different from date in line 48.

  - [Register here](https://sites.google.com/sheffield.ac.uk/ai-research-day/register).

#### Upcoming External Events

- [UKRI National Compute Ecosystem Webinar](https://events.teams.microsoft.com/event/8ee13916-4bce-489f-942e-5d9fca034c75@8bb7e08e-daa4-4a8e-927e-fca38db04b7e)
  - Date: **2026-08-06**
  - Following the UKRI National Compute Ecosystem Town Hall meeting in London on 2nd June 2026, UKRI is running this webinar to provide an opportunity for more people in the research and innovation community to engage with us on our plans for the Next National Supercomputing Service (NNSS) and the National Compute Resources (NCRs). In this webinar, we will be sharing the slides that were presented in the town hall meeting and a panel Q&A, so there will be time for attendees to share their thoughts and ask questions.
- [Ihaka Lecture Series 2026](https://www.eventbrite.co.nz/e/ihaka-lecture-series-2026-tickets-1991653429020)
  - Date: **2026-08-08**
  - Title: Spotting the Weird Ones
  - Speaker: Professor Rob J Hyndman, Monash University, Australia
- [SDE Forge - Secure Data Environments Technical Skills Residential](https://sde-forge.eventbrite.co.uk/)

>   - Dates: **2026-11-11 to 2026-11-13**

Reviewer 2: ```suggestion
  - Dates: **2026-11-11 - 2026-11-13**
```

Consistency with above events

  - Venue: The Rookery Hotel, Crewe
  - This workshop will focus on the Research Technical Professional (RTP) role and the skills needed to design, build and maintain Secure Data Environments (SDEs).
- [Computing Insight UK 2026](https://www.sc.stfc.ac.uk/ciuk-2026/)

>   - Dates: **2026-12-02 to 2026-12-04**

Reviewer 2: ```suggestion
  - Dates: **2026-12-02 - 2026-12-04**
```

Consistency

  - Closing date : **2026-08-10**
- [Research Software Engineer](https://careers.tudelft.nl/job/Delft-Research-Software-Engineer-2628-CD/1364153857/)
  - Closing date : **2026-08-26**
  - Our [Open Science NL](https://www.openscience.nl/) project, OPynSim, is looking for a Research Software Engineer (RSE)! The project aims to develop Python bindings to an existing C++ codebase (OpenSim) and develop new file formats (+ associated APIs, viewers, etc.) for the ecosystem. If you're interested in designing/developing/maintaining projects like OPynSim/OpenSim, and think the idea of developing tools in C++/Python sounds interesting then check it out!

---

### Community

#### Digital Research Practice Support Community

The DRPS community is a group for people that support researchers in carrying out research in the digital age. Meetings
are held monthly, with discussions around events, training and opportunities related to the field.

You can join the google group
[here](https://groups.google.com/u/1/a/sheffield.ac.uk/g/digital-research-practice-support-community-group/about) to
stay informed.

> The next meeting is scheduled for 2pm on Wednesday 15th August 2026.

Reviewer: ```suggestion
The next meeting is scheduled for 2pm on Wednesday 16th September 2026.
```
